homeassistant/components/netatmo/sensor.py

```diff
@@ -145,17 +157,9 @@ def _retry(_data):
                 module_items = config[CONF_MODULES]
                 for module_name in module_items:
                     if module_name not in data.get_module_names():
+                        _LOGGER.info("Module %s not found", module_name)
                         continue
-                    for condition in data.station_data.monitoredConditions(
-                            module_name):
-                        dev.append(
-                            NetatmoSensor(
-                                data,
-                                module_name,
-                                condition.lower(),
-                                data.station
-                            )
-                        )
+                    dev.extend(find_devices(data))
```

Comment:
Did you mean to call with `module_items` as the second argument here?

Reply:
Duh... Yes of course. Thanks for the hint.